### PR TITLE
[docs] Improve clarity around storage of states by wallets.

### PR DESCRIPTION
**Pull request by geoknee**
Following discussion in our forum.

### CODE REVIEW

File: packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md [hunk 1]
```diff
@@ -188,6 +188,14 @@ There is also an optimization where a state can be supported by `n` signatures o
 
 In the extreme, this allows a single state signed by all `n` parties to be accepted by the chain.
 
+:::note
+In most cases where a support proof is required for some change of state of the chain, the entire proof is submitted in the call data, and no on chain `State` is involved. The [`respond`](./clear-a-challenge#call-respond) method is an exception to this rule, and allows for the submission of only a single state in certain circumstances, with the support proof being implied by a combination of on-chain and submitted data.
```
Comment (snario) [inline packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md:192]: > the entire proof is submitted in the call data
I'd keep the term "call data" as one word, `calldata`

> no on chain `State` is involved
You should use the term "on-chain storage"

> a combination of on-chain and submitted data.
A combination of on-chain storage and data provided in calldata

File: packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md [hunk 2]
```diff
@@ -188,6 +188,14 @@ There is also an optimization where a state can be supported by `n` signatures o
...
+:::
+
+:::tip
+Nitro wallets need only store the "last" `n` states, because they never need to submit more than `n` states to the chain.
```
Comment (snario) [inline packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md:196]: Do we define "support" and "supported" state in an easily linkable way here? We should reference that part of the docs if so

File: packages/nitro-protocol/website/versioned_docs/version-0.1.1/wallet-dev-tutorial/execute-state-transitions.md [hunk 3]
```diff
@@ -183,12 +183,20 @@ States with turn numbers `n` through `2n-1` (inclusive) are known as "post fund
 
 Although you can check the validity of a state transition by providing the data above to an on-chain method, to cause any meaningful change in on-chain state (such as releasing funds), digitial signatures on those states are also required.
 
-Nitro protocol uses the idea of supporting a state: in order for the chain to accept a channel state, `s`, that channel state must be _supported_ by `n` signatures (where `n = participants.length`). The simplest way for this to accomplish this is to provide a sequence of `n` states terminating is state `s`, where each state is signed by its mover and each consecutive pair of states form a valid transition.
+Nitro protocol uses the idea of **supporting** a state: in order for the chain to accept a channel state, `s`, that channel state must be _supported_ by `n` signatures (where `n = participants.length`). The simplest way for this to accomplish this is to provide a sequence of `n` states terminating is state `s`, where each state is signed by its mover and each consecutive pair of states form a valid transition.
```
Comment (snario) [inline packages/nitro-protocol/website/versioned_docs/version-0.1.1/wallet-dev-tutorial/execute-state-transitions.md:186]: 👍

File: packages/nitro-protocol/website/versioned_docs/version-0.1.1/wallet-dev-tutorial/execute-state-transitions.md [hunk 4]
```diff
@@ -183,12 +183,20 @@ States with turn numbers `n` through `2n-1` (inclusive) are known as "post fund
...
 There is also an optimization where a state can be supported by `n` signatures on a sequence of `m < n` states, provided again that each consecutive pair of those `m` states form a valid transition and further provided each participant has provided a signature on a state later or equal in the sequence than the state for which they were the mover.
 
 In the extreme, this allows a single state signed by all `n` parties to be accepted by the chain.
 
+:::note
+In most cases where a support proof is required for some change of state of the chain, the entire proof is submitted in the call data, and no on chain `State` is involved. The [`respond`](./clear-a-challenge#call-respond) method is an exception to this rule, and allows for the submission of only a single state in certain circumstances, with the support proof being implied by a combination of on-chain and submitted data.
+:::
+
+:::tip
+Nitro wallets need only store the "last" `n` states, because they never need to submit more than `n` states to the chain.
```
Comment (snario) [inline packages/nitro-protocol/website/versioned_docs/version-0.1.1/wallet-dev-tutorial/execute-state-transitions.md:197]: Nitro wallet? State Channels wallet? Wallets that support the Nitro Protocol?

File: packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md (hunk 1 above)
Comment (geoknee) [reply packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md:192]: > You should use the term "on-chain storage"

To be precise, on-chain storage is checked (`largestTurnNum`). It's just that no validTransition check is done.

File: packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md (hunk 2 above)
Comment (geoknee) [reply packages/nitro-protocol/docs/wallet-dev-tutorial/execute-state-transitions.md:196]: It's a couple paragraphs north,  in this section.

**Pull request merged**